src/platforms/javascript/common/session-replay/index.mdx

```diff
@@ -21,6 +11,8 @@ description: "Learn how to enable Session Replay in your app if it is not alread
 
 <Include name="beta-note-session-replay.mdx" />
 
+Sentry's Replay product is geared toward developers. By default, it'll mask all fiends, so no personally identifiable information (PII) is collected, eliminating any concerns with General Data Protection Regulation (GDPR). Learn more on <PlatformLink to="/session-replay/privacy">Session Replay Privacy</PlatformLink>.
```

Comment:
Isn't Sentry geared towards developers? Is there something more specific you want to say here?
```suggestion
Sentry's Replay product is geared toward developers. By default, it'll mask all fields, so no personally identifiable information (PII) is collected, eliminating any concerns with General Data Protection Regulation (GDPR). Learn more on <PlatformLink to="/session-replay/privacy">Session Replay Privacy</PlatformLink>.
```